src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -200,6 +200,12 @@ public static void ValidateIssuerSecurityKey(SecurityKey securityKey, SecurityTo
                     throw LogHelper.LogExceptionMessage(new SecurityTokenInvalidSigningKeyException(LogHelper.FormatInvariant(LogMessages.IDX10232, securityKey)){ SigningKey = securityKey });
             }
 
+            if (validationParameters.RequireSignedTokens == false && securityKey == null)
+            {
+                LogHelper.LogInformation(LogMessages.IDX10252);
+                return;
+            }
+
             if (securityKey == null)
```

Comment:
We need a exception here that explains the securityKey is null but validationParameters.RequireSignedTokens == true. This indicates an error, but I have to admit, it would be hard to get here.

Reply:
addressed with 822afd1165f5f6323218bda6534bdd3f97cc4d46